website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -91,18 +91,13 @@ rendering must include a [`consul` block](/nomad/docs/job-specification/consul)
 #### OIDC login with PKCE
```

Comment:
In [my release notes PR](https://github.com/hashicorp/nomad/pull/25433), I moved this section out of upgrade specific and into the release notes proper.  If we need to duplicate that content between release notes and upgrade details, we should put that content into a partial in the release notes PR.

Reply:
oh! I was wondering how I missed that - it's not merged yet!

I'm happy to remove this from here, but I think it'll cause a merge conflict with yours whether I remove it or change it. Which would you prefer? (I would prefer not to touch it again here, so I can avoid dismissing Tim's approval, but not a big deal.)

Reply:
Ok. I'll copy the change you made here to my PR. Then deal with the merge conflict after your PR is merged. I am planning to merge the release notes PR on Tuesday.

Reply:
alrighty, thank you! <3